### DISCUSSION TITLE
Using PyCharm

### DISCUSSION THREAD

**Discussion by antigraviton** [reflex-dev/reflex#131] [Q&A]
I wanted to give it a go but ran into trouble as soon as I opened the (demo) app in PyCharm.

Everything after:
```
def index():
  return = pc.center(
    ...
  )
```
is marked with a warning that 'center' is not callable.

This makes it unusable using my preferred IDE, which is a significant obstacle for me.

Ideas? Do I need to do/set something in PyCharm?

**Comment by picklelo** [comment]
This may be related to [this ticket](https://github.com/pynecone-io/pynecone/issues/54). I think the problem is we do some Python magic when creating these component functions that Pycharm may not be picking up correctly. We will aim to fix this soon, thanks for reporting.
